src/main/resources/citizen/templates/document/claimIssueReceipt.html

```diff
@@ -141,9 +141,8 @@
   <table>
     <tr>
       <td rowspan="6">
-        <img
```

Comment:
is img tag removed?

Reply:
no, just replaced by the most recent one.